Sources/PrimerSDK/Classes/Data Models/PrimerSettings.swift

```diff
@@ -82,10 +82,12 @@ public class PrimerApplePayOptions: PrimerApplePayOptionsProtocol {
     
     let merchantIdentifier: String
     let merchantName: String
+    let isCaptureBillingAddressEnabled: Bool
     
-    public init(merchantIdentifier: String, merchantName: String) {
+    public init(merchantIdentifier: String, merchantName: String, isCaptureBillingAddressEnabled: Bool) {
```

Comment:
Can you provide a default value so we don't break the API?

Reply:
Weeeird I thought I did for the exact same purpose 😓 
Thanks for that 💪